packages/autorest.python/samples/specification/directives/generated/azure/directives/sample/_operations/_operations.py

```diff
@@ -206,11 +240,12 @@ def begin_basic_polling(
         """
 
     @distributed_trace
-    def begin_basic_polling(self, product: Optional[Union[JSON, IO]] = None, **kwargs: Any) -> CustomPoller[JSON]:
+    def begin_basic_polling(self, product: Optional[Union[JSON, JSON, IO]] = None, **kwargs: Any) -> CustomPoller[JSON]:
```

Comment:
we shouldn't have json, json

Reply:
yes, reverted this miss commit. Changes in this PR is only for `models-mode==dpg` generation